apb.yml

```diff
@@ -8,7 +8,7 @@ tags:
   documentationUrl: "https://docs.aerogear.org/external/apb/metrics.html"
   providerDisplayName: "Red Hat, Inc."
   sdk-docs-android: "https://docs.aerogear.org/external/apb/metrics/android.html"
```

Comment:
can we also remove the docs link here. Just keep documentationUrl and remove the rest